src/ManageCourses.Api/Data/DataService.cs

```diff
@@ -174,6 +157,30 @@ public void ProcessPayload(Payload payload)
                         Name = organisation.Name
                     }
                     );
+            }            
+            foreach (var institution in payload.Institutions)
+            {
+                _context.AddUcasInstitution(
+                    new UcasInstitution
```

Comment:
Maybe we should consider auto mapper

Reply:
Possibly, but I don't want to do this in this PR